requirements/requirements.txt

```diff
@@ -2,13 +2,16 @@ accelerate
 asyncio
 deepspeed>=0.14.0
 deepspeed-kernels
+fastapi
+fastchat
 Flask-RESTful
 grpcio
 grpcio-tools
 Pillow
-pydantic
+pydantic==1.*
```

Comment:
FYI @adk9 - we're trying to get the [Pydantic update](!423) merged, hopefully very soon.  So I was curious on what issues you ran into that you needed to pin this?

Reply:
> FYI @adk9 - we're trying to get the [Pydantic update](https://github.com/microsoft/DeepSpeed-MII/pull/!423) merged, hopefully very soon.

Oh great, thanks for the pointer! I was just starting to look into this myself.

> So I was curious on what issues you ran into that you needed to pin this?

I see a `400 Bad Request` error. 
```
Error code: 400 - {'object': 'error', 'message': "[{'type': 'missing', 'loc': ('query', 'request'), 'msg': 'Field required', 'input': None}]", 'code': 40001}
```
I think the issue is with fastapi not fully supporting working with pydantic.v1 while having pydantic v2 installed (https://github.com/tiangolo/fastapi/issues/10360). So the workaround is to either pin fastapi or pydantic to an older version, and I chose to do the latter.

Reply:
Resolved as we have updated to pydantic 2